operations/network/management_network/aruba/index_aruba.md

```diff
@@ -4,7 +4,7 @@
 
 ### ARUBA - CX Platform based networks
 
-![](../img/intro.png)
+![](../../../../../../img/network/management_network/network/management_networkintro.png)
```

Comment:
Seems to be no such file anywhere in the branch... Also, amount of "../" seems excessive.

Reply:
fixed